back/engines/commercial/public_api/app/controllers/public_api/v2/ideas_controller.rb

```diff
@@ -25,5 +50,16 @@ def finder_params
         .to_h
         .symbolize_keys
     end
+
+    def permitted_idea_params
+      params.require(:idea).permit(
+        :assignee_id,
+        :idea_status_id,
+        title_multiloc: {},
+        body_multiloc: {},
+        topic_ids: [],
+        phase_ids: [],
```

Comment:
Debated a bit how to do this but in the end decided to go with the way admins create ideas in the main app. Happy to hear thoughts, though.